docs/src/pages/components/skeleton.mdx

```diff
@@ -0,0 +1,132 @@
+
+Text Skeletons inherit their `display` property because in some situations it is needed to format
+the text properly (e.g. `h1`'s are `block`s, and spans are `inline` and this makes a difference to
+text's `line-height`). If the parent isn't a typographical element, you may need to define the
+`display` prop manually to get it to look just the way you want.
+
+---
+
+### Wrapping
+
+The second way to use the `Skeleton` is by wrapping an element that you want to cover. For instance,
+if you had a fixed size `img` that was still loading, you could wrap it with a Skeleton and it'd
+simply take the shape of your image and hide it.
+
+```tsx live
+<Component initialState={{ loading: true }}>
+    {({state, setState}) =>
+        <section>
+            <div style={{paddingBottom: '1rem'}}>
+                <Skeleton loading={state.loading}>
+                    <img src={withPrefix('/images/anchor-logo.svg')} />
+                </Skeleton>
+            </div>
+
+            <Button onClick={() => setState({loading: !state.loading})}>
+                Toggle Skeleton
+            </Button>
+        </section>
+    }
+</Component>
+```
+
+If the wrapped child has a `border-radius`, you'll probably want to give the the `Skeleton` the same
+`border-radius` (especially if the child is circular, like
+an <Link to="/components/avatar">`Avatar`</Link>).
+
+The `Skeleton` does not know the `display` setting of the child it's wrapping, so you may need to
+set it manually (i.e. if the child is a `block` you'll probably want to set the `Skeleton` to a
+`block` as well, so that it fits in the DOM the same way).
+
+---
+
+### Custom Shape
+
+You might just want to give the Skeleton specific dimensions to make it take the exact shape you desire.
+
+```tsx live
+<section style={{whiteSpace: 'normal'}}>
+    <Skeleton width="100%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+    <Skeleton width="100%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+    <Skeleton width="80%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+</section>
+```
+
+<br />
+
+---
+
+## API
+
+<ApiTable data={[
+    {
+        property: 'loading',
+        description: `It is assumed that the <pre>Skeleton</pre> is being used as content is being
+            loaded. As such, if <pre>loading</pre> is set to <pre>true</pre> then the
+            <pre>Skeleton</pre> is visible. When <pre>loading</pre> is set to <pre>false</pre> the
+            <pre>Skeleton</pre> is hidden and its children are revealed.`,
+        type: 'boolean',
+        default: 'true',
+    },
+    {
+        property: 'textLength',
+        description: `The amount of text to replace with Skeleton blocks. This is useful for when
+            you don't know how long the text will be, or if you don't have dummy-text ala Lorem
+            Ipsum. Note that settings this prop will give it precedence over the content it
+            contains.`,
+        type: 'number',
+    }
```

Comment:
We should probably add the `children<string>` in the API to outline how it treats that text.

Reply:
That's a fair point. This is one of the few components where the child is important.